cpe775/utils/img_utils.py

```diff
@@ -1,12 +1,22 @@
-import matplotlib.pyplot as plt
 import numpy as np
+import torchvision.transforms.functional as F
 from PIL import Image
-from skimage import io
 from torchvision import utils
-import torchvision.transforms.functional as F
 
 import cv2
 
+# from skimage import io
+# import matplotlib.pyplot as plt
+# importing these is casing segmentation fault
+# so, we changed the backend
+try:
```

Comment:
Isn't it better try to import the matplotlib without changing the backend and then if the script raises an exception try to change the backend?
```python
try:
  import matplotlib.pyplot as plt
  from skimage import io
except ImportError
  import matplitlib as mpl
  mlp.use('TkAgg')
  import matplotlib.pyplot as plt
  from skimage import io
```

Reply:
It would be better that way!
However the segmentation fault doesn't raise an Error but a Signal.